### PR TITLE
UW-553 JSON Schema for JEDI driver

### PR DESCRIPTION
**Pull request by maddenp-noaa**
**Synopsis**

Add JSON Schema support for JEDI driver.

**Type**

- [x] Enhancement (adds a new functionality)

**Impact**

- [x] This is a non-breaking change (existing functionality continues to work as expected)

**Checklist**

- [x] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

### CODE REVIEW

File: src/uwtools/drivers/jedi.py [hunk 1]
```diff
@@ -117,11 +117,9 @@ def validate_only(self):
...
         config = self.configuration_file()
         yield [executable, config]
-        cmd = "%s && time %s --validate-only %s 2>&1" % (
-            " && ".join(self._driver_config["execution"]["envcmds"]),
-            refs(executable),
-            refs(config),
-        )
+        cmd = "time %s --validate-only %s 2>&1" % (refs(executable), refs(config))
+        if envcmds := self._driver_config["execution"].get("envcmds"):
+            cmd = " && ".join([*envcmds, cmd])
```
Comment (maddenp-noaa) [inline src/uwtools/drivers/jedi.py:122]: `envcmds` is optional, but we were treating it as if it were required. This makes it optional again, prepending `cmd` with a `&&`-connected pipeline of all defined commands.

File: docs/shared/chgres_cube.yaml [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
     mpiargs:
-      - --export=NONE
+      - "--export=NONE"
     mpicmd: srun
     threads: 1
   namelist:
     update_values:
       config:
-        atm_files_input_grid: gfs.t{{ cycle.strftime('%h') }}z.atmanl.nemsio
+        atm_files_input_grid: atm.t{{cycle.strftime('%H') }}z.nc
```
[the remark this answers is not in the document]
Comment (christinaholtNOAA) [reply docs/shared/chgres_cube.yaml:11]: I think this is likely fine. It's good to keep both the `.nc` options for the `atm` and `sfc` components.

**Review by christinaholtNOAA — APPROVED** [review]
LGTM!

**Review by NaureenBharwaniNOAA — APPROVED** [review]
LGTM!

**Review by elcarpenterNOAA — APPROVED** [review]
LGTM.

**Pull request merged**